README.md

```diff
@@ -274,6 +274,9 @@ assertHint(R.id.edittext, "Hint");
 ```java
 assertError(R.id.edittext, R.string.error);
 assertError(R.id.edittext, "Error message");
+
+assertNoError(R.id.edittext, R.string.error);
```

Comment:
Does it make sense to pass an error message to the assertion?

If I want to check that there are no errors, I would use `assertNoError()`. If I want to check that the error is "Error B" I would use `assertError("Error A")`, not `assertNoError("Error B")`.

Reply:
Done, removed